### PR TITLE
RELATED: BB-1850 removed customizable button

### PR DESCRIPTION
**Pull request by strouhaljan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/170/ : SUCCESS in 1m 06s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/48/ : SUCCESS in 6s

File: docs/measure_value_filter_component.md [hunk 1]
```diff
@@ -49,67 +50,56 @@ const locationResort = Model.attribute(
...
+| measureIdentifier | true      | string                                                     | The identifier of the filtered measure. You can use either the local identifier or URI.                                                                                                                                |
+| onApply           | true      | Function                                                   | A callback when the selection is confirmed by a user. The passed configuration of the measure value filter is already transformed into a measure value filter definition, which you can then send directly to a chart. |
+| onCancel          | true      | Function                                                   | A callback when user clicks on the Cancel button or invokes dropdown to be closed by clicking outside of it. Use this callback to hide dropdown.                                                                       |
```
Comment (evgeniabogranskaya) [inline docs/measure_value_filter_component.md]: A callback when a user clicks the Cancel button or invokes the dropdown to be closed by clicking outside of it. Use this callback to hide the dropdown.

File: docs/measure_value_filter_component.md [hunk 2]
```diff
@@ -49,67 +50,56 @@ const locationResort = Model.attribute(
...
+| filter            | false     | [Filter](filter_visual_components.md#measure-value-filter) | The measure value filter definition                                                                                                                                                                                    |
+| displayDropdown   | false     | boolean                                                    | Specifies whether the dropdown is opened                                                                                                                                                                               |
+| anchorEl          | false     | event target or string                                     | Element which dropdown is aligned to, typically your toggle button                                                                                                                                                     |
```
Comment (evgeniabogranskaya) [inline docs/measure_value_filter_component.md]: The element that the dropdown is aligned to, typically your toggle button

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/182/ : SUCCESS in 1m 29s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/53/ : SUCCESS in 18s

**Pull request merged**